src/uhs/atomizer/atomizer/state_machine.hpp

```diff
@@ -45,6 +45,12 @@ namespace cbdc::atomizer {
         [[nodiscard]] auto commit(nuraft::ulong log_idx, nuraft::buffer& data)
             -> nuraft::ptr<nuraft::buffer> override;
 
+        /// Handler for the raft cluster configuration changes
+        /// \param log_idx Raft log number of the configuration change.
+        void commit_config(
```

Comment:
I think we should add this to all the state machines, not just atomizer.

Reply:
Agreed.